_data/icons.yml

```diff
@@ -0,0 +1,177 @@
+# Icons on the footer
+# Usage: Modify 'id' for the icon to show up.
+behance:
+  pre: "https://www.behance.net/" 
+  id: "sam" # e.g. "sam"
+  pos:
+  icon:
```

Comment:
Nah that is a pain to add as an icon,
I'd rather have it done with the code

Reply:
With the change I proposed above (but did not implement) the user would only have to put the desired icons on a yaml collection on a `_config`. Similar thing could be made to 'share icons'.

Reply:
done